src/pages/error-handling.tsx

```diff
@@ -52,3 +59,14 @@ export const ErrorFallback = () => {
     </MainContent>
   );
 };
+
+export const ErrorCard = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <Card>
+      <CardSection>
+        ⚠ Unexpected error, please contact us on GitHub!
+        <Box mt={2}>{children}</Box>
+      </CardSection>
+    </Card>
+  );
+};
```

Comment:
A generic component to display a basic error message inside a "card".

We use a font character `⚠` for the warning mark rather than an icon component to make it very robust 😄 .